Shared/Models/Message Model/Messageable.swift

```diff
@@ -40,8 +40,7 @@ protocol Messageable {
     var recentReplies: [Messageable] { get }
     var lastUpdatedAt: Date? { get }
 
-    var emotionCounts: [Emotion : Int] { get }
-    var expression: Expression? { get }
+    var expressions: [Expression] { get }
```

Comment:
I combined emotions and expressions. This frees up reactions for reads. However I dont see a way to "update" or "add" an attachment after its been sent,  so I may need to reconsider this approach.

Reply:
It may actually be best to use this approach for sending, and reactions for non author expressions. We would just include a URL in extraData of the reaction.

Reply:
But the downside is read reactions, may not work correctly.